src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -4808,6 +4941,57 @@ namespace HeatBalanceManager {
 			PerformSolarCalculations();
 		}
 
+		// Initialize zone outdoor environmental variables
+		// Bulk Initialization for Temperatures & WindSpeed
+		// using the zone, modify the zone  Dry/Wet BulbTemps
+		SetZoneOutBulbTempAt();
+		CheckZoneOutBulbTempAt();
+
+		// X Luo Added 07/19/2017
+		// set zone level wind dir to global value
+		SetZoneWindSpeedAt();
+		SetZoneWindDirAt();
+
+		// X Luo Added 07/30/2017
+		// Set zone data to linked air node value if defined.
+		GetOutAirNodesInput();
+		for ( ZoneNum = 1; ZoneNum <= NumOfZones; ++ZoneNum ) {
+			if ( Zone( ZoneNum ).HasLinkedOutAirNode ) {
+				if ( Node( Zone( ZoneNum ).LinkedOutAirNode ).SchedOutAirDryBulb ) {
```

Comment:
Does this loop *always* execute?  If there aren't any SurfaceProperty:LocalEnvironment objects this should be skipped.  Same question down in the surface heat balance.  Seems like there should be a global for AnyLocalEnvironmentsInModel similar to AnyEnergyManagementSystemInModel.